experimental/framework/tinylicious-client/src/interfaces.ts

```diff
@@ -31,6 +31,11 @@ export interface TinyliciousContainerServices {
     audience: ITinyliciousAudience;
 }
 
+export interface TinyliciousContainerAndServices {
```

Comment:
Thoughts on an alternate name here instead of "ContainerAndServices"? Otherwise, the name may become outdated if we add on another value to this object, alongside the container and services.
Perhaps "TinyliciousAssets" or "TinyliciousRuntime"? @ChumpChief any suggestions?

Reply:
Or TinyliciousCore

Reply:
@skylerjokiel had a great suggestion here of "TinyliciousResources". I feel like that's a good term for aggregating the container and services together in one object, and being flexible for any future additions